submissions/description/insufficient_security_configurability/verification_of_contact_method_not_required/template.md

```diff
@@ -0,0 +1,21 @@
+# Lack of Verification Email
+
+## Overview of the Vulnerability
+
+The overall security of an application is diminished when accounts are not properly configured to include a notification email upon contact method account changes, such as a phone number or email address change. A lack of a verification email for an updated contact method allows a malicious attacker to associate any email address or phone number with an account without verification. This could allow them to perform spam, phishing, or impersonation attacks.
```

Comment:
The description of this vulnerability is not accurate, this is actually used as a substitute for things such as Emails not being verified on user account creation and similar submissions.

> The overall security of an application is diminished when accounts are not properly verified upon creation of a new contact method, allowing an attacker to associate their account with an unvalidated method of contact. The lack of a verification email for a contact method allows an attacker to associate a contact method with an account without verification, This allows them to perform phishing, or impersonation attacks, and can lead to attacks such as account squatting.